balancer-labs/frontend-v2
# Feature - Pool locking module

# Description

Adds a module to the 80/20 WETH/BAL lockable pool similar to the staking pool, which allows users to get a quick glance at their lockable status.

## Type of change


## How should this be tested?

Open up. the 80/20 WETH/BAL pool and then see if you can see the module. On connected wallet should show locked and unlocked LP tokens as well as test expiry of lock.

## Visual context

<img width="465" alt="image" src="https://user-images.githubusercontent.com/40786269/182160111-67576d3b-5de9-4678-9589-b4ebb3946d20.png">


## Checklist:
